recipes/ligo-requests/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "ligo-requests" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "5575ab76aa00ad175b1493ab53db809e1327f423fc8004798daf45bab1748bd5"
```

I think the update bot doesn't work if you quote the checksums:
```suggestion
  sha256: 5575ab76aa00ad175b1493ab53db809e1327f423fc8004798daf45bab1748bd5
```
The only bit that generally needs to be quoted is the version. It doesn't matter with `X.Y.Z` versioning but it does with two digits such as `1.1` and `1.10` as they get interpreted as YAML floats and the trailing `0` is dropped.